watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py

```diff
@@ -182,51 +182,100 @@ def display_results(m):
+    )
+    print(f"\nTotal Capital Costs: {total_capital_cost:.3f} $")
+    print(f"Normalized Capital Costs: {normalized_capex:.3f} $/m3/hr")
+    print("Capital Cost Breakdown")
     for u in m.fs.costing._registered_unit_costing:
         print(
             u.name,
             " : {price:0.3f} $".format(
                 price=value(pyunits.convert(u.capital_cost, to_units=pyunits.USD_2018))
             ),
         )
-    print("\nUtility Costs\n")
+
+    total_operating_cost = value(
+        pyunits.convert(
+            m.fs.costing.total_operating_cost, to_units=pyunits.USD_2018 / pyunits.year
+        )
+    )
+    print(f"\nTotal Operating Cost: {total_operating_cost:.3f} $/year")
+
+    opex_fraction = total_operating_cost / (
+        total_operating_cost
+        + value(m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor)
+    )
+    print(f"Operating cost fraction: {opex_fraction:.3f} $ opex / $ annual")
+
+    print("Operating Cost Breakdown")
     for f in m.fs.costing.flow_types:
         print(
-            f,
-            " :    {price:0.3f} M$/year".format(
+            f.title(),
+            " :    {price:0.3f} $/m3 feed".format(
                 price=value(
                     pyunits.convert(
                         m.fs.costing.aggregate_flow_costs[f],
-                        to_units=pyunits.MUSD_2018 / pyunits.year,
+                        to_units=pyunits.USD_2018 / pyunits.year,
+                    )
+                    / pyunits.convert(
+                        m.fs.feed.flow_vol[0],
+                        to_units=pyunits.m**3 / pyunits.year,
                     )
                 )
             ),
         )
 
-    print("")
-    total_capital_cost = value(
-        pyunits.convert(m.fs.costing.total_capital_cost, to_units=pyunits.MUSD_2018)
-    )
-    print(f"Total Capital Costs: {total_capital_cost:.3f} M$")
-    total_operating_cost = value(
+    electricity_intensity = value(
         pyunits.convert(
-            m.fs.costing.total_operating_cost, to_units=pyunits.MUSD_2018 / pyunits.year
+            m.fs.costing.electricity_intensity, to_units=pyunits.kWh / pyunits.m**3
         )
     )
-    print(f"Total Operating Costs: {total_operating_cost:.3f} M$/year")
-    electricity_intensity = value(
+    print(f"\nElectricity Intensity: {electricity_intensity:.3f} kWh/m^3")
+
+    sec = electricity_intensity * value(
         pyunits.convert(
-            m.fs.costing.electricity_intensity, to_units=pyunits.kWh / pyunits.m**3
+            m.fs.product_struvite.flow_mass_comp[0, "struvite"],
+            to_units=pyunits.kg / pyunits.s,
+        )
+        / pyunits.convert(
+            m.fs.feed.properties[0].flow_vol, to_units=pyunits.m**3 / pyunits.s
         )
     )
-    print(f"Electricity Intensity: {electricity_intensity:.3f} kWh/m^3")
+    print(f"Specific energy consumption: {sec:.3f} kWh/m3 feed")
```

Comment:
I'm confused here. `electricity_intensity` has units kWh/m3. Then `sec` is `electricity_intensity` times struvite mass flow rate divided by mass flowrate. So that has units of (kWh/m3) * (kg/s) / (m3/s) [=] kWh x kg / m6. But line 273 states units of kWh/m3.

Am I missing something here, or is there a mistake?

Reply:
Thanks for this catch. The flows should be inverted and displayed on a per kg struvite basis. sec = kWh/m3 * (m3/s) / (kg/s) = kWh/kg. I will update this shortly.

Reply:
I think the specific energy consumption is an input, right? So why you want to display it as a result in the report?

Reply:
In this case, we are considering the energy input of the unit as well as the pump, normalized on a per kg struvite basis.